src/Transition/CreateNewIDFUsingRulesV9_6_0.f90

```diff
@@ -386,6 +386,14 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
               ! If your original object starts with A, insert the rules here
 
               ! If your original object starts with C, insert the rules here
+
+              CASE('CONTROLLER:MECHANICALVENTILATION')
+                CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                nodiff=.false.
+                OutArgs(1:CurArgs)=InArgs(1:CurArgs)
+                IF (MakeUPPERCase(OutArgs(4)) == "VENTILATIONRATEPROCEDURE") THEN
+                  OutArgs(4) = "Standard62.1VentilationRateProcedure"
+                ENDIF
```

Comment:
Transition rule for changing the `Controller:MechanicalVentilation`'s `System Outdoor Air Method` from `VentilationRateProcedure` to `Standard62.1VentilationRateProcedure`.